app/grandchallenge/evaluation/migrations/0069_phase_maximum_settable_memory_gb_evaluation_and_more.py

```diff
@@ -24,7 +24,7 @@ class Migration(migrations.Migration):
             model_name="phase",
             name="evaluation_selectable_gpu_type_choices",
             field=models.JSONField(
-                default=grandchallenge.evaluation.models.get_default_gpu_type_choices,
+                default=grandchallenge.components.utils.get_default_gpu_type_choices,
```

Comment:
When I run `makemigrations`, the move of this function is not detected. 

According to the [Django docs](https://docs.djangoproject.com/en/5.1/topics/migrations/#:~:text=References%20to%20functions%20in%20field%20options%20such%20as%20upload_to%20and%20limit_choices_to%20and%20model%20manager%20declarations%20with%20managers%20having%20use_in_migrations%20%3D%20True%20are%20serialized%20in%20migrations%2C%20so%20the%20functions%20and%20classes%20will%20need%20to%20be%20kept%20around%20for%20as%20long%20as%20there%20is%20a%20migration%20referencing%20them.): 
> References to functions in field options [...] are serialized in migrations, so the functions and classes will need to be kept around for as long as there is a migration referencing them. 

I therefore edited this manually. However, I think this might lead to some issues?